### PR TITLE
Reorganize topology checks description and images

### PR DESCRIPTION
**Pull request by havatv**
Make sure that the illustrations for the topology checks in digitizing options are placed with the explaining text by using a table of images (avoid using `.. figure` for these).

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -3526,4 +3512,7 @@ other one.
    :width: 1.5em
 .. |toggleEditing| image:: /static/common/mActionToggleEditing.png
    :width: 1.5em
+.. |gapcheck| image:: img/gapcheck.png
+.. |overlapcheck| image:: img/overlapcheck.png
+.. |missingvertexcheck| image:: img/missingvertexcheck.png
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: In its current shape (may require @SrNetoChan confirmation), if the scripts/find_set_subst.py is run, these substitutions would be erased and the build broken. Any added substitution should be added in the substitution.txt file. And I'm not sure it's worth adding substitutions used once.

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -3333,33 +3333,19 @@ features. Since other features may be present in the same area, topological
...
-
-.. figure:: img/missingvertexcheck.png
-   :align: center
-
-
+.. list-table::
+
+   * - The |checkbox| :guilabel:`Gap` check will check for
+       gaps between neighbouring polygons.
+     - |gapcheck|
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Replace this with `.. figure:: img/gapcheck.png` (or `.. image...`)? We should not need one-shot substitution.

Comment (havatv) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: Would have to be `.. image`.

File: source/docs/user_manual/working_with_vector/vector_properties.rst (hunk 1 above)
Comment (havatv) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: I have removed the substitutions. They are not necessary when I use "list-table".

**Comment by havatv** [comment]
Header row added.

File: source/docs/user_manual/working_with_vector/vector_properties.rst (hunk 2 above)
Comment (havatv) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: Replacement done.

**Comment by havatv** [comment]
Thanks for the review, @DelazJ !

**Pull request merged**